base/docs/basedocs.jl

```diff
@@ -3185,14 +3185,24 @@ Any
 """
     Union{}
 
-`Union{}`, the empty [`Union`](@ref) of types, is the type that has no values. That is, it has the defining
-property `isa(x, Union{}) == false` for any `x`. `Base.Bottom` is defined as its alias and the type of `Union{}`
-is `Core.TypeofBottom`.
+`Union{}`, the empty [`Union`](@ref) of types, is the *bottom* type of the type system. That is, for each
+`T::Type`, `Union{} <: T`. Also see the subtyping operator's documentation: [`<:`](@ref).
+
+As such, `Union{}` is also an *empty*/*uninhabited* type, meaning that it has no values. That is, for each `x`,
+`isa(x, Union{}) == false`.
+
+`Base.Bottom` is defined as its alias and the type of `Union{}` is `Core.TypeofBottom`.
 
 # Examples
 ```jldoctest
 julia> isa(nothing, Union{})
 false
+
+julia> Union{} <: Int
+true
+
+julia> typeof(Union{}) === Core.TypeofBottom
+true
```

Comment:
Maybe also show
```
julia> isa(Union{}, Union)
false
```
here? But I wonder whether all the discussion of `Bottom` shouldn't go into the extended help section. It's a bit in the weeds.

Reply:
In particular, it's something you need to know if you're doing type system manipulation, but it's not really something you need to know if you're just wondering what `Union{}` is.

Reply:
IMO an extended help section isn't necessary for such a tiny doc string?